lib/components/molecules/Form/FieldInput/tests/FieldInput.test.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { shallowWithTheme, renderWithTheme } from '../../../../../../__mock__/themeMock';
+import { shallow } from 'enzyme';
+import { renderWithTheme } from '../../../../../../__mock__/themeMock';
 import { FieldInputVanilla } from '../FieldInput';
 
 describe('FieldInput Component', () => {
   let FieldInput = '';
 
   test.only('should render correctly', () => {
-    FieldInput = shallowWithTheme(<FieldInputVanilla />);
```

Comment:
Why make this change, it will throw error for undefined theme without it

Reply:
Just testing some thing :)